docker/puppetdb/Dockerfile

```diff
@@ -65,48 +40,25 @@ LABEL org.label-schema.maintainer="Puppet Release Team <release@puppet.com>" \
       org.label-schema.schema-version="1.0" \
       org.label-schema.dockerfile="/Dockerfile"
 
-# Values from /etc/default/puppetdb
-ENV JAVA_BIN="/usr/bin/java"
-ENV USER="puppetdb"
-ENV GROUP="puppetdb"
-ENV INSTALL_DIR="/opt/puppetlabs/server/apps/puppetdb"
-ENV CONFIG="/etc/puppetlabs/puppetdb/conf.d"
-ENV BOOTSTRAP_CONFIG="/etc/puppetlabs/puppetdb/bootstrap.cfg"
-ENV SERVICE_STOP_RETRIES=60
+COPY --from=build /puppetdb.deb /puppetdb.deb
+
+RUN wget http://nightlies.puppet.com/apt/puppet6-nightly-release-"$UBUNTU_CODENAME".deb && \
+    dpkg -i puppet6-nightly-release-"$UBUNTU_CODENAME".deb && \
+    rm puppet6-nightly-release-"$UBUNTU_CODENAME".deb && \
+    apt-get update && \
+    apt-get install --no-install-recommends -y /puppetdb.deb && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
 
 COPY docker/puppetdb/logback.xml /etc/puppetlabs/puppetdb/logback.xml
 COPY docker/puppetdb/request-logging.xml /etc/puppetlabs/puppetdb/request-logging.xml
 COPY docker/puppetdb/conf.d /etc/puppetlabs/puppetdb/conf.d/
-COPY resources/puppetlabs/puppetdb/bootstrap.cfg /etc/puppetlabs/puppetdb/
-RUN mkdir -p $LOGDIR
-
-RUN addgroup $GROUP && adduser --system $USER --ingroup $GROUP
-
-ADD https://raw.githubusercontent.com/puppetlabs/pupperware/8cdc056be09c44ed1ba96f963d71bd074d063b17/shared/ssl.sh /ssl.sh
-RUN chmod +x /ssl.sh
-
-ADD https://raw.githubusercontent.com/puppetlabs/wtfc/6aa5eef89728cc2903490a618430cc3e59216fa8/wtfc.sh /wtfc.sh
-RUN chmod +x /wtfc.sh
-COPY docker/puppetdb/docker-entrypoint.sh /
-RUN chmod +x /docker-entrypoint.sh
-COPY docker/puppetdb/docker-entrypoint.d /docker-entrypoint.d
-
-EXPOSE 8080 8081
-
-ENTRYPOINT ["/usr/bin/tini", "-g", "--", "/docker-entrypoint.sh"]
-CMD ["services"]
-
-COPY docker/puppetdb/healthcheck.sh /
-RUN chmod +x /healthcheck.sh
-# The start-period describes how long it takes PuppetDB to come
-# up in the worst case (LCOW). The other timing parameters are set so that it
-# takes at most 2 minutes to realize that PuppetDB has failed.
-# Probe failure during --start-period will not be counted towards the maximum number of retries
-HEALTHCHECK --start-period=3m --interval=10s --timeout=10s --retries=12 CMD ["/healthcheck.sh"]
+COPY resources/puppetlabs/puppetdb/bootstrap.cfg /etc/puppetlabs/puppetdb/bootstrap.cfg
```

Comment:
This was added for LCOW per the commit message... but the behavior seems to be different between Travis and Azure - we should figure out why.

Reply:
This was also related to the symlinks in the git repo causing issues on windows. Should be resolved by updated `COPY`ing